docs/configuration.md

```diff
@@ -655,6 +659,13 @@ interpretation of the options.
 | `privatekey` | yes   | The private key for Cloudfront, provided by AWS.        |
 | `keypairid` | yes    | The key pair ID provided by AWS.                         |
 | `duration` | no      | An integer and unit for the duration of the Cloudfront session. Valid time units are `ns`, `us` (or `µs`), `ms`, `s`, `m`, or `h`. For example, `3000s` is valid, but `3000 s` is not. If you do not specify a `duration` or you specify an integer without a time unit, the duration defaults to `20m` (20 minutes).|
+|`ipfilteredby`|no|A string with the following value `none|aws|awsregion`. |
+|`awsregion`|no|a comma separated string of AWS regions, only available when `ipfilteredby` is `awsregion`. For example, `us-east-1, us-west-2`|
```

Comment:
Can these be autodetected through the host?

Reply:
yes, we could autodetect the region of the host node from AWS meta API. But what happened if the distribution container is deployed in another region which is different from where S3 is hosted?